en_US/data-integration/data-bridge-mqtt.md

```diff
@@ -0,0 +1,340 @@
+        replayq {offload = false, seg_bytes = "100MB"}
+        retry_interval = "1s"
+        server = "192.168.1.234:1883"
+        ssl {
+          ciphers = ["TLS_AES_256_GCM_SHA384", "TLS_AES_128_GCM_SHA256"]
+          depth = 10
+          enable = false
+          reuse_sessions = true
+          secure_renegotiate = true
+          user_lookup_fun = "emqx_tls_psk:lookup"
+          verify = "verify_none"
+          versions = ["tlsv1.3", "tlsv1.2", "tlsv1.1", "tlsv1"]
+        }
+        username = "user1"
+      }
+      direction = "ingress"
+      enable = true
+      local_topic = "local/topic/ingress"
+      local_qos = 0
+      remote_qos = 0
+      remote_topic = "remote/topic/ingress"
+      payload = "${payload}"
+      retain = false
+    }
+    mqtt_bridge_egress {
+      connector {
+        bridge_mode = false
+        clean_start = true
+        keepalive = "60s"
+        max_inflight = 32
+        mode = "cluster_shareload"
+        password = "emqx"
+        proto_ver = "v4"
+        reconnect_interval = "15s"
+        replayq {offload = false, seg_bytes = "100MB"}
+        retry_interval = "15s"
+        server = "192.168.1.234:1883"
+        ssl {
+          ciphers = ["TLS_AES_256_GCM_SHA384", "TLS_AES_128_GCM_SHA256"]
+          depth = 10
+          enable = false
+          reuse_sessions = true
+          secure_renegotiate = true
+          user_lookup_fun = "emqx_tls_psk:lookup"
+          verify = "verify_none"
+          versions = ["tlsv1.3", "tlsv1.2", "tlsv1.1", "tlsv1"]
+        }
+        username = "emqx"
+      }
+      direction = "egress"
+      enable = true
+      local_topic = "local/topic/egress"
+      remote_qos = 0
+      remote_topic = "remote/topic/egress"
+      payload = "${payload}"
+      retain = false
+    }
+  }
+}
+```
+
+Open EMQX Dashboard and click `Data Integration` - `Data Bridges` on the right side to see the two bridges created.
+![image](./assets/rules/mqtt_bridge/dashboard_show_bridges.png)
+
+::: tip
+Note that the 5.0.0 Dashboard does not support detachment rules at this time, using the MQTT Bridge alone, which can only be created using the configuration file
+:::
+
+### The `ingress bridge` message flow
+
+```txt
+ +-------------------------+
+ | Remote                  |          +--------+
+ | EMQX Broker             |<---------| Client |
+ |                         |          +--------+
+ +-------------------------+
+             |
+             |
+             V
+  +----------------------+
+  |  MQTT Bridge Ingress |
+  +----------------------+
+             |
+             V
+  +------------------------+
+  | Local                  |          +--------+
+  | EMQX Broker            |--------->| Client |
+  |                        |          +--------+
+  +------------------------+
+
+```
+
+Using the desktop MQTT client MQTTX, create two connections, one to the local and one to the external service. And subscribe to `local/topic/ingress` locally.
```

Comment:
```suggestion
Using the desktop MQTT client MQTTX, create two connections: one to the local and one to the external service. Then subscribe to `local/topic/ingress` locally.
```
Also regarding the last sentence, it's not very clear what does it mean "subscribe locally"